source/partials/next_release/_release.md.erb

```diff
@@ -1,15 +1,29 @@
 #### API Enhancements
 
-* <%= link_to_commit 'f47d054171b07e2ce9c2efa9e02dde6979fc9c8f', "#{link_to_api 'encryption'} to encrypt secrets.", 1829 %>
+- <%= link_to_commit '73595c1d18f7455c5b7aac39dce319c987f0c231', 'Check Package Connection API (#3048)', 3048 %>
```

Comment:
You can additionally use `link_to_api` to add a link to the API docs. We do this for a couple of reasons —

* to improve SEO
* to help users discover links to the various sites we have